web/src/components/organisms/FilePickerInput .jsx

```diff
@@ -0,0 +1,13 @@
+import { FileInput, Label } from 'flowbite-react';
+import React from 'react';
+
+
+const FilePickerInput  = ({ multiple = false }) => {
+    return (
+        <div className="mb-2"> 
+            <FileInput id="default-file-upload" multiple={multiple} />
+        </div>
```

Comment:
not all who use this component needs mb-2, so apply mb-2 on where this is called
```suggestion
        <> 
            <FileInput id="default-file-upload" multiple={multiple} />
        </>
```